client/layout/masterbar/publish.jsx

```diff
@@ -101,5 +107,8 @@ class MasterbarItemNew extends React.Component {
 }
 
 export default connect( state => {
-	return { selectedSite: getSelectedSite( state ) };
+	return {
+		selectedSite: getSelectedSite( state ),
+		recordTracksEvent,
```

Comment:
Note that this is in `mapStateToProps` but needs to be dispatched so belongs in `mapDispatchToProps`. I also recommend hiding `recordTracksEvent` from the component API and instead choose something more semantically meaningful to the component to expose, such as `closeDrafts` or `onCloseDrafts`